layouts/shortcodes/resource_template_add_file_only_if_ruby.md

```diff
@@ -5,6 +5,6 @@ a file based on a template, and then use Ruby to specify a condition:
 template '/tmp/somefile' do
   mode '0755'
   source 'somefile.erb'
-  only_if do ! File.exist?('/etc/passwd') end
+  only_if { ! ::File.exist?('/etc/passwd') }
```

Comment:
Also this is going to alert in the near future. Why not just use not_if here and avoid the need to negate the value returned?